modulus/models/diffusion/preconditioning.py

```diff
@@ -780,10 +777,14 @@ def forward(
         c_in = 1 / (self.sigma_data**2 + sigma**2).sqrt()
         c_noise = sigma.log() / 4
 
-        x = torch.cat((x, img_lr), dim=1)
+        if img_lr is None:
```

Comment:
Can we change the name `img_lr` to `image_conditioning` or similar? It need not just be low resolution...could be e.g. topography.